openedx/core/djangoapps/course_live/permissions.py

```diff
@@ -41,8 +44,14 @@ def has_permission(self, request, view):
         if GlobalStaff().has_user(request.user):
             return True
 
+        permissions = [
+            CourseRolesPermission.VIEW_ALL_CONTENT.value,
+            CourseRolesPermission.VIEW_ONLY_LIVE_PUBLISHED_CONTENT.value,
+            CourseRolesPermission.VIEW_ALL_PUBLISHED_CONTENT.value
+        ]
         return (
             CourseInstructorRole(course_key).has_user(request.user) or
             CourseStaffRole(course_key).has_user(request.user) or
+            course_permissions_list_check_any(request.user, permissions, course_key) or
```

Comment:
I think you didn't push the definition of this function

Reply:
https://github.com/openedx/edx-platform/pull/33419/files#diff-913b2aa09cc1220394f787380476c9e5531aac9c8bf54fb88359b9a38ca73ff3